tests/checks/test_SimpleCheck.py

```diff
@@ -0,0 +1,110 @@
+import re
+from unittest import TestCase
+import mock
+
+from pandas import DataFrame
+
+from shift_detector.checks.simple_check import SimpleCheck
+from shift_detector.precalculations.store import Store
+
+
+class TestSimpleCheck(TestCase):
+
+    def setUp(self):
+        sales1 = {'shift': ['A'] * 100, 'no_shift': ['C'] * 100}
+        sales2 = {'shift': ['B'] * 100, 'no_shift': ['C'] * 100}
+        self.df1 = DataFrame.from_dict(sales1)
+        self.df2 = DataFrame.from_dict(sales2)
+        self.store = Store(self.df1, self.df2)
+        self.check = SimpleCheck()
+
+    def test_init(self):
+        print("###### \n\n\nParameters missing\n\n\n######")
+        self.assertTrue(True)
+        # with self.subTest("Test wrong columns"):
+        #     # self.assertRaises(TypeError, lambda: DistinctionCheck(['shift', 0]))
+        #
+        # with self.subTest("Test wrong num epochs"):
+        #     # self.assertRaises(TypeError, lambda: DistinctionCheck(num_epochs='wrong'))
+        #     # self.assertRaises(ValueError, lambda: DistinctionCheck(num_epochs=0))
+        #
+        # with self.subTest("Test wrong relative threshold"):
+        #     self.assertRaises(TypeError, lambda: DistinctionCheck(relative_threshold='wrong'))
+        #     self.assertRaises(ValueError, lambda: DistinctionCheck(relative_threshold=-1))
+
+    def test_run_categorical(self):
+        with self.subTest("Test precalculation"):
+            report = self.check.run(self.store)
+
+        with self.subTest("Test shifted categorical columns"):
+            self.assertEqual(report.shifted_columns, ['shift'])
+            self.assertCountEqual(report.examined_columns, ['shift', 'no_shift'])
+            self.assertEqual(report.explanation['shift'], 'Attribute: A with Diff: 1.0\n')
+
+    def test_run_numerical(self):
+        with self.subTest("Test precalculation"):
+            numbers1 = {'cool_numbers': [1, 2, 3, 4]*10}
+            numbers2 = {'cool_numbers': [1, 2, 3, 6]*10}
+            self.df1 = DataFrame.from_dict(numbers1)
```

Comment:
Do this instead and delete lines 46, 47

```suggestion
            self.df1 = DataFrame({'cool_numbers': [1, 2, 3, 4] * 10})
```

Reply:
deprecated